homeassistant/components/aftership/sensor.py

```diff
@@ -181,8 +181,8 @@ async def async_update(self, **kwargs):
                 track["tracking_number"] if track["title"] is None else track["title"]
             )
             last_checkpoint = (
-                "Shipment pending"
-                if track["tag"] == "Pending"
+                f"Shipment {track['tag'].lower()}"
```

Comment:
by doing this, it will not be a breaking change since "pending" is one of the tags it can return.